vet.sh

```diff
@@ -41,7 +41,7 @@ if [[ "$1" = "-install" ]]; then
   popd
   if [[ -z "${VET_SKIP_PROTO}" ]]; then
     if [[ "${GITHUB_ACTIONS}" = "true" ]]; then
-      PROTOBUF_VERSION=25.2 # a.k.a. v4.22.0 in pb.go files.
```

Comment:
Oof.  Maybe just change this to `v4.x.y` so that it doesn't have to be kept in sync, if you don't mind?

Reply:
This should not be changed. The release version 25.2 uses v4.22.0 as the version number in the generated pb.go files

Reply:
Ah, I did not understand! I assumed that someone had updated the PROTOBUF_VERSION without updating the comment next to it. I'll revert this, thanks.